[0m00:30:20.495365 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f996c5c12a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f996b7421d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f996b742170>]}


============================== 00:30:20.498414 | 7563ca7b-a3ae-4fa4-800d-573ee905c65a ==============================
[0m00:30:20.498414 [info ] [MainThread]: Running with dbt=1.9.3
[0m00:30:20.499066 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'debug': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m00:30:26.470821 [debug] [MainThread]: Starter project path: /home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/lib/python3.10/site-packages/dbt/include/starter_project
[0m00:30:26.476569 [info ] [MainThread]: 
Your new dbt project "dbt_pipeline" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m00:30:26.477341 [info ] [MainThread]: Setting up your profile.
[0m00:32:53.757272 [info ] [MainThread]: Profile dbt_pipeline written to /home/ubuntu/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m00:32:53.763023 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 153.33713, "process_in_blocks": "0", "process_kernel_time": 0.220147, "process_mem_max_rss": "108464", "process_out_blocks": "80", "process_user_time": 1.606568}
[0m00:32:53.765016 [debug] [MainThread]: Command `dbt init` succeeded at 00:32:53.764558 after 153.34 seconds
[0m00:32:53.766772 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f996c5c12a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f996b743010>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f996b7429e0>]}
[0m00:32:53.768428 [debug] [MainThread]: Flushing usage events
[0m00:32:56.176327 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m01:05:44.316079 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1872ecd2a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f187204a1d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f187204a170>]}


============================== 01:05:44.318414 | e049334a-6efa-4fab-a2d9-97bc1a0ad4f5 ==============================
[0m01:05:44.318414 [info ] [MainThread]: Running with dbt=1.9.3
[0m01:05:44.319133 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m01:05:44.326324 [info ] [MainThread]: dbt version: 1.9.3
[0m01:05:44.326855 [info ] [MainThread]: python version: 3.10.12
[0m01:05:44.327372 [info ] [MainThread]: python path: /home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/bin/python3
[0m01:05:44.327801 [info ] [MainThread]: os info: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.35
[0m01:05:44.513577 [info ] [MainThread]: Using profiles dir at /home/ubuntu/.dbt
[0m01:05:44.514200 [info ] [MainThread]: Using profiles.yml file at /home/ubuntu/.dbt/profiles.yml
[0m01:05:44.514750 [info ] [MainThread]: Using dbt_project.yml file at /home/ubuntu/airflow/dbt_project.yml
[0m01:05:44.515392 [error] [MainThread]: Encountered an error:
Internal Error
  Profile should not be None if loading profile completed
[0m01:05:44.516492 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_success": false, "command_wall_clock_time": 0.25575477, "process_in_blocks": "0", "process_kernel_time": 0.11942, "process_mem_max_rss": "109316", "process_out_blocks": "16", "process_user_time": 1.423095}
[0m01:05:44.516929 [debug] [MainThread]: Command `dbt debug` failed at 01:05:44.516823 after 0.26 seconds
[0m01:05:44.517276 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1872ecd2a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f18720a2b30>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f1870e56ce0>]}
[0m01:05:44.517722 [debug] [MainThread]: Flushing usage events
[0m01:05:45.856575 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m01:07:21.014183 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7cece152a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7cebf921a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f7cebf92140>]}


============================== 01:07:21.016659 | a7587eb0-f46d-4e63-a767-a87b2db28673 ==============================
[0m01:07:21.016659 [info ] [MainThread]: Running with dbt=1.9.3
[0m01:07:21.017179 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m01:07:21.025049 [info ] [MainThread]: dbt version: 1.9.3
[0m01:07:21.025635 [info ] [MainThread]: python version: 3.10.12
[0m01:07:21.026112 [info ] [MainThread]: python path: /home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/bin/python3
[0m01:07:21.026697 [info ] [MainThread]: os info: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.35
[0m01:07:21.215192 [info ] [MainThread]: Using profiles dir at /home/ubuntu/.dbt
[0m01:07:21.215690 [info ] [MainThread]: Using profiles.yml file at /home/ubuntu/.dbt/profiles.yml
[0m01:07:21.216230 [info ] [MainThread]: Using dbt_project.yml file at /home/ubuntu/airflow/dbt_project.yml
[0m01:07:21.216897 [info ] [MainThread]: adapter type: snowflake
[0m01:07:21.217325 [info ] [MainThread]: adapter version: 1.9.2
[0m01:07:21.217953 [info ] [MainThread]: Configuration:
[0m01:07:21.218407 [info ] [MainThread]:   profiles.yml file [[32mOK found and valid[0m]
[0m01:07:21.218865 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m01:07:21.219456 [info ] [MainThread]: Required dependencies:
[0m01:07:21.220233 [debug] [MainThread]: Executing "git --help"
[0m01:07:21.222717 [debug] [MainThread]: STDOUT: "b"usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m01:07:21.223299 [debug] [MainThread]: STDERR: "b''"
[0m01:07:21.223686 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m01:07:21.224184 [info ] [MainThread]: Connection:
[0m01:07:21.224692 [info ] [MainThread]:   account: https://pd73040.us-east-2.aws.snowflakecomputing.com
[0m01:07:21.225155 [info ] [MainThread]:   user: ARSHAVIN
[0m01:07:21.225649 [info ] [MainThread]:   database: dbt_db
[0m01:07:21.226346 [info ] [MainThread]:   warehouse: dbt_wh
[0m01:07:21.227099 [info ] [MainThread]:   role: dbt_role
[0m01:07:21.228501 [info ] [MainThread]:   schema: dbt_schema
[0m01:07:21.229543 [info ] [MainThread]:   authenticator: None
[0m01:07:21.230393 [info ] [MainThread]:   oauth_client_id: None
[0m01:07:21.231823 [info ] [MainThread]:   query_tag: None
[0m01:07:21.232429 [info ] [MainThread]:   client_session_keep_alive: False
[0m01:07:21.232904 [info ] [MainThread]:   host: None
[0m01:07:21.233415 [info ] [MainThread]:   port: None
[0m01:07:21.233869 [info ] [MainThread]:   proxy_host: None
[0m01:07:21.234287 [info ] [MainThread]:   proxy_port: None
[0m01:07:21.234623 [info ] [MainThread]:   protocol: None
[0m01:07:21.235172 [info ] [MainThread]:   connect_retries: 1
[0m01:07:21.235555 [info ] [MainThread]:   connect_timeout: None
[0m01:07:21.235888 [info ] [MainThread]:   retry_on_database_errors: False
[0m01:07:21.236216 [info ] [MainThread]:   retry_all: False
[0m01:07:21.236604 [info ] [MainThread]:   insecure_mode: False
[0m01:07:21.236973 [info ] [MainThread]:   reuse_connections: True
[0m01:07:21.237508 [info ] [MainThread]: Registered adapter: snowflake=1.9.2
[0m01:07:21.304150 [debug] [MainThread]: Acquiring new snowflake connection 'debug'
[0m01:07:21.324858 [debug] [MainThread]: Using snowflake connection "debug"
[0m01:07:21.325255 [debug] [MainThread]: On debug: select 1 as id
[0m01:07:21.325599 [debug] [MainThread]: Opening a new connection, currently in state init
[0m01:09:57.778628 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3d96df12a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3d95f721d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3d95f72170>]}


============================== 01:09:57.780983 | 0ddd6309-fa06-45e7-b7f0-489d3cd76f26 ==============================
[0m01:09:57.780983 [info ] [MainThread]: Running with dbt=1.9.3
[0m01:09:57.781680 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'debug': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m01:10:05.449418 [error] [MainThread]: Encountered an error:

[0m01:10:05.463329 [error] [MainThread]: Traceback (most recent call last):
  File "/home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/lib/python3.10/site-packages/dbt/cli/requires.py", line 153, in wrapper
    result, success = func(*args, **kwargs)
  File "/home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/lib/python3.10/site-packages/dbt/cli/requires.py", line 103, in wrapper
    return func(*args, **kwargs)
  File "/home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/lib/python3.10/site-packages/dbt/cli/main.py", line 477, in init
    results = task.run()
  File "/home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/lib/python3.10/site-packages/dbt/task/init.py", line 324, in run
    project_name = self.get_valid_project_name()
  File "/home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/lib/python3.10/site-packages/dbt/task/init.py", line 278, in get_valid_project_name
    name = click.prompt("Enter a name for your project (letters, digits, underscore)")
  File "/home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/lib/python3.10/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
  File "/home/ubuntu/dbt_snowflake_airflow/dbtairflow_venv/lib/python3.10/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m01:10:05.467702 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": false, "command_wall_clock_time": 7.7486706, "process_in_blocks": "0", "process_kernel_time": 0.049622, "process_mem_max_rss": "91812", "process_out_blocks": "16", "process_user_time": 1.240553}
[0m01:10:05.469806 [debug] [MainThread]: Command `dbt init` failed at 01:10:05.469306 after 7.75 seconds
[0m01:10:05.471602 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3d96df12a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3d95f73190>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3d95fc5060>]}
[0m01:10:05.473477 [debug] [MainThread]: Flushing usage events
[0m21:40:20.372896 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3c029687f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3c01aea2c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3c01aea320>]}


============================== 21:40:20.376734 | 824d5c39-4648-4969-bd06-5c5d21578728 ==============================
[0m21:40:20.376734 [info ] [MainThread]: Running with dbt=1.9.3
[0m21:40:20.377338 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run -s raw_stg_orders', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m21:40:20.377859 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/ubuntu/airflow/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m21:40:20.378774 [debug] [MainThread]: Resource report: {"command_name": "run", "command_success": false, "command_wall_clock_time": 0.0661361, "process_in_blocks": "8", "process_kernel_time": 0.08054, "process_mem_max_rss": "95668", "process_out_blocks": "16", "process_user_time": 1.228236}
[0m21:40:20.379362 [debug] [MainThread]: Command `dbt run` failed at 21:40:20.379251 after 0.07 seconds
[0m21:40:20.379773 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3c029687f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3c01ae9600>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f3c01aea2c0>]}
[0m21:40:20.380194 [debug] [MainThread]: Flushing usage events
[0m21:40:21.712049 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m21:40:30.485417 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa82a3f47f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa82a4b1d80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa8295636d0>]}


============================== 21:40:30.488631 | eb69bf57-fa8b-4f86-a481-12ba0346c78e ==============================
[0m21:40:30.488631 [info ] [MainThread]: Running with dbt=1.9.3
[0m21:40:30.489577 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/ubuntu/.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt build', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m21:40:30.490366 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/ubuntu/airflow/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m21:40:30.491950 [debug] [MainThread]: Resource report: {"command_name": "build", "command_success": false, "command_wall_clock_time": 0.0676305, "process_in_blocks": "0", "process_kernel_time": 0.0602, "process_mem_max_rss": "95756", "process_out_blocks": "0", "process_user_time": 1.254172}
[0m21:40:30.492773 [debug] [MainThread]: Command `dbt build` failed at 21:40:30.492625 after 0.07 seconds
[0m21:40:30.493339 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa82a3f47f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa8297b38b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fa8297b3d60>]}
[0m21:40:30.494075 [debug] [MainThread]: Flushing usage events
[0m21:40:31.559029 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m23:49:36.016153 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcd4b878640>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcd4a9f6350>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcd4a9f63b0>]}


============================== 23:49:36.018612 | 4290697f-aa15-40f4-8437-c334bbb5ce1d ==============================
[0m23:49:36.018612 [info ] [MainThread]: Running with dbt=1.9.3
[0m23:49:36.019144 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': '/home/ubuntu/.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt build', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m23:49:36.019663 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/ubuntu/airflow/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m23:49:36.020771 [debug] [MainThread]: Resource report: {"command_name": "build", "command_success": false, "command_wall_clock_time": 0.063201204, "process_in_blocks": "0", "process_kernel_time": 0.140746, "process_mem_max_rss": "95680", "process_out_blocks": "8", "process_user_time": 1.065648}
[0m23:49:36.022844 [debug] [MainThread]: Command `dbt build` failed at 23:49:36.022648 after 0.07 seconds
[0m23:49:36.023341 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcd4b878640>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcd4a9f5690>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7fcd4a9f6350>]}
[0m23:49:36.023756 [debug] [MainThread]: Flushing usage events
[0m23:49:37.484345 [debug] [MainThread]: An error was encountered while trying to flush usage events
